Datahub.Core/Layout/Topbar.razor

```diff
@@ -131,7 +131,7 @@
     {
         try {
             User = await UserInformationService.GetUserAsync();
-            _email = Configuration["Contact:Email"] ?? "datahub@nrcan-rncan@gc.ca";
+            _email = Configuration[$"{ConfigurationConstants.CONTACT_EMAIL}"] ?? "datahub@nrcan-rncan@gc.ca";
```

Comment:
Default email has a typo. It might be a good idea to use a constant for the default-in-code email as well as the config key.

Reply:
Good catch. ty